api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceServiceImpl.java

```diff
@@ -197,17 +197,31 @@ public List<WorkspaceResponse> getFeaturedWorkspaces() {
   }
 
   @Override
-  public List<WorkspaceResponse> listWorkspacesAsService() {
```

Comment:
removed because it was non-performant and we don't need this - only the namespace.  A major component of the runtime was mapping the RWB entity object.